misk/src/main/kotlin/misk/tracing/backends/jaeger/JaegarSamplerConfig.kt

```diff
@@ -2,6 +2,10 @@ package misk.tracing.backends.jaeger
 
 import misk.config.Config
 
+/**
+ * Configuration for Jaeger's sampler. If values are left null then Jaeger will provide defaults.
+ * See [com.uber.jaeger.Configuration.SamplerConfiguration] for default values.
```

Comment:
are the default values public? if so, might be nice to inline and remove the nullability

Reply:
Not all of them which is super annoying. That was my initial plan until I noticed difference levels of visibility. 

Also would've been nice if they had public builders for their configs too which would have made instantiating them a little less gross.

:-/